<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>398</width>
    <height>436</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Set Graph Property</string>
  </property>
  <widget class="QDialogButtonBox" name="confirmButtonBox">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>380</y>
     <width>341</width>
     <height>32</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
   <property name="standardButtons">
    <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
   </property>
  </widget>
  <widget class="QWidget" name="verticalLayoutWidget">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>20</y>
     <width>341</width>
     <height>61</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QFrame" name="frame_2">
      <property name="frameShape">
       <enum>QFrame::Box</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Plain</enum>
      </property>
      <widget class="QLabel" name="label_3">
       <property name="geometry">
        <rect>
         <x>30</x>
         <y>10</y>
         <width>56</width>
         <height>12</height>
        </rect>
       </property>
       <property name="text">
        <string>X min</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
      <widget class="QLineEdit" name="Xmin">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>30</y>
         <width>81</width>
         <height>21</height>
        </rect>
       </property>
      </widget>
      <widget class="QLabel" name="label_4">
       <property name="geometry">
        <rect>
         <x>130</x>
         <y>10</y>
         <width>56</width>
         <height>12</height>
        </rect>
       </property>
       <property name="text">
        <string>X max</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
      <widget class="QLineEdit" name="Xmax">
       <property name="geometry">
        <rect>
         <x>120</x>
         <y>30</y>
         <width>81</width>
         <height>21</height>
        </rect>
       </property>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="verticalLayoutWidget_2">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>80</y>
     <width>341</width>
     <height>61</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout_2">
    <item>
     <widget class="QFrame" name="frame_4">
      <property name="frameShape">
       <enum>QFrame::Box</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Plain</enum>
      </property>
      <widget class="QLabel" name="label_7">
       <property name="geometry">
        <rect>
         <x>30</x>
         <y>10</y>
         <width>56</width>
         <height>12</height>
        </rect>
       </property>
       <property name="text">
        <string>Y min</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
      <widget class="QLineEdit" name="Ymin">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>30</y>
         <width>81</width>
         <height>21</height>
        </rect>
       </property>
      </widget>
      <widget class="QLabel" name="label_8">
       <property name="geometry">
        <rect>
         <x>130</x>
         <y>10</y>
         <width>56</width>
         <height>12</height>
        </rect>
       </property>
       <property name="text">
        <string>Y max</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
      <widget class="QLineEdit" name="Ymax">
       <property name="geometry">
        <rect>
         <x>120</x>
         <y>30</y>
         <width>81</width>
         <height>21</height>
        </rect>
       </property>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QFrame" name="frame_5">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>150</y>
     <width>341</width>
     <height>81</height>
    </rect>
   </property>
   <property name="frameShape">
    <enum>QFrame::Box</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Raised</enum>
   </property>
   <property name="lineWidth">
    <number>3</number>
   </property>
   <widget class="QRadioButton" name="R">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>20</y>
      <width>81</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>R</string>
    </property>
    <attribute name="buttonGroup">
     <string notr="true">dtypebuttonGroup</string>
    </attribute>
   </widget>
   <widget class="QRadioButton" name="T">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>40</y>
      <width>81</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>T</string>
    </property>
    <attribute name="buttonGroup">
     <string notr="true">dtypebuttonGroup</string>
    </attribute>
   </widget>
   <widget class="QRadioButton" name="S1">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>20</y>
      <width>81</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>S1</string>
    </property>
    <attribute name="buttonGroup">
     <string notr="true">dtypebuttonGroup</string>
    </attribute>
   </widget>
   <widget class="QRadioButton" name="S2">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>40</y>
      <width>81</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>S2</string>
    </property>
    <attribute name="buttonGroup">
     <string notr="true">dtypebuttonGroup</string>
    </attribute>
   </widget>
   <widget class="QRadioButton" name="E1">
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>20</y>
      <width>81</width>
      <height>16</height>
     </rect>
    </property>
    <property name="contextMenuPolicy">
     <enum>Qt::DefaultContextMenu</enum>
    </property>
    <property name="text">
     <string>E1</string>
    </property>
    <attribute name="buttonGroup">
     <string notr="true">dtypebuttonGroup</string>
    </attribute>
   </widget>
   <widget class="QRadioButton" name="E2">
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>40</y>
      <width>81</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>E2</string>
    </property>
    <attribute name="buttonGroup">
     <string notr="true">dtypebuttonGroup</string>
    </attribute>
   </widget>
  </widget>
  <widget class="QListWidget" name="dataListWidget">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>240</y>
     <width>151</width>
     <height>101</height>
    </rect>
   </property>
   <property name="selectionMode">
    <enum>QAbstractItemView::ExtendedSelection</enum>
   </property>
   <property name="sortingEnabled">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="addplotButton">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>260</y>
     <width>21</width>
     <height>23</height>
    </rect>
   </property>
   <property name="cursor">
    <cursorShape>ArrowCursor</cursorShape>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="icon">
    <iconset resource="../resource/Arrow.qrc">
     <normaloff>:/icon/Arrow.png</normaloff>:/icon/Arrow.png</iconset>
   </property>
   <property name="autoDefault">
    <bool>true</bool>
   </property>
   <property name="flat">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="removeplotButton">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>300</y>
     <width>21</width>
     <height>23</height>
    </rect>
   </property>
   <property name="cursor">
    <cursorShape>ArrowCursor</cursorShape>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="icon">
    <iconset resource="../resource/Arrow.qrc">
     <normaloff>:/icon/Arrow_mirror.png</normaloff>:/icon/Arrow_mirror.png</iconset>
   </property>
   <property name="autoDefault">
    <bool>true</bool>
   </property>
   <property name="flat">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QListWidget" name="plotListWidget">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>240</y>
     <width>151</width>
     <height>101</height>
    </rect>
   </property>
   <property name="selectionMode">
    <enum>QAbstractItemView::ExtendedSelection</enum>
   </property>
   <property name="sortingEnabled">
    <bool>true</bool>
   </property>
  </widget>
 </widget>
 <resources>
  <include location="../resource/Arrow.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>confirmButtonBox</sender>
   <signal>accepted()</signal>
   <receiver>Dialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>confirmButtonBox</sender>
   <signal>rejected()</signal>
   <receiver>Dialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <buttongroups>
  <buttongroup name="dtypebuttonGroup"/>
 </buttongroups>
</ui>
